MIP61/MIP61.md

```diff
@@ -105,10 +105,16 @@ In practice, the Governance Facilitators will calculate compensation for Recogni
 
 #### Performance Modifier
 
-Governance Facilitators track participation and communication metrics for each Recognized Delegate. For the purpose of calculating the Performance Modifier, we look at the minimum of the two metrics. For example, if a Recognised Delegate has 85% participation and 80% communication then the Performance Modifier is based on `min(85%, 80%) = 80%`.
+Governance Facilitators track participation and communication metrics for each Recognized Delegate. For the purpose of calculating the Performance Modifier, we look at the minimum of the two metrics. For example, if a Recognised Delegate has 85% participation and 80% communication then the Performance Modifier is based on `min(85%, 80%) = 80%`. A few parameters are used to determine the effect of the Performance Modifier.
 
-- For less than 75%, the Recognised Delegate will receive no compensation.
-- For 75% or above, the Performance Modifier starts at 40% and rises linearly to 100% for metrics of 90% and above.
+| Parameter | Math Symbol | Initial Value |
+| --------- | :-----: | ----: |
+| Threshold for poor performance | L | 75% |
+| Threshold for adequate performance | H | 90% |
```

Comment:
I'm not sold on poor and adequate, maybe change poor to adequate, so above 75 is adequate, and then change above 90 to good?

Reply:
Sure, whatever naming you want is fine.